docs/css/content.css

```diff
@@ -124,13 +124,14 @@
 #content #output-header {
   margin: 1.25em 0 0 0; border: none; padding: .25em 1.2em .25em 1.2em;
   font-size: 80%;
-  color: white; background-color: silver;
+  color: black;
+  background-color: #f0f0f0;
 }
 #content #output {
   overflow: auto;
   max-height: 20em;
   margin: 0; padding: .5em 1em; border: 2px solid silver; border-top: none;
-  background-color: #f0f0f0;
+  background-color: silver;
```

Comment:
Why exchange `#f0f0f0` and `silver`? :)

Reply:
I made the output colored, and it was easier to change the backgrounds so that yellow integers showed up than it was to figure out a different color for integers.  I'm willing to turn colored output back off, or change the output colors if you'd prefer.

Reply:
No, I meant that in the other place you changed `silver` to `#f0f0f0` :)

Reply:
Anyway, it just seemed odd. If it was intentional, don't mind my comment

Reply:
I would use `#c0c0c0` for readability though.

Reply:
You mean `#c0c0c0` instead of `silver`, right?

Reply:
Yes